pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -88,6 +88,8 @@
 
   <ItemGroup>
     <IgnoredReference Include="System.Private.CoreLib" />
+    <IgnoredReference Include="System.Threading.Overlapped" />
```

Comment:
I see, let's file an issue to remove them.  This will impact testing.

private.corelib is intentionally missing, these two are bugs.  I don't want to get in the habit of suppressing missing dependencies since the consequences downstream are harder for folks to diagnose.